docs/guides/gutenberg.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Gutenberg"
+menu:
+  main:
+    parent: "guides"
+---
+## Using Gutenberg in with Timber
+Timber works with Gutenberg out of the box. If you use `{{ post.content }}`, Timber will render all the Gutenberg blocks.
+
+## ACF Blocks
+### What are the ACF Blocks
+ACF Blocks are the alternative way to create content blocks without advanced JavaScript knowledge. If you want to learn more about them read the article on [advancedcustomfields.com](https://www.advancedcustomfields.com/blog/acf-5-8-introducing-acf-blocks-for-gutenberg/).
+
+### How to use ACF Blocks with Timber
+Before you can start using ACF Blocks, you must install the Advanced Custom Fields 5.8.0-beta or later.
+
+To create your content block, first you have to register it in `functions.php` or in a seperate plugin:
+```php
+add_action('acf/init', 'my_acf_init');
+function my_acf_init() {
+	
+	// check function exists
+	if( function_exists('acf_register_block') ) {
+		
+		// register a new block
+		acf_register_block(array(
+			'name'				=> 'example_block',
+			'title'				=> __('Example Block'),
+			'description'		=> __('A custom example block.'),
+			'render_callback'	=> 'my_acf_block_render_callback',
+			'category'			=> 'formatting',
+			'icon'				=> 'admin-comments',
+			'keywords'		    => array( 'example' ),
+		));
+	}
+}
+```
+
+next you you have to create a `render_callback`:
+```php
+function my_acf_block_render_callback( $block ) {
+    // store block values
+    $vars['block'] = $block;
+
+    // store field values
+    $vars['fields'] = get_fields(); 
+
+    // render the block
+    Timber::render( '/template-parts/block/example-block.twig', $vars );
+}
+
+```
+you create an extra array called `$vars` with two values:
+- **block** - with all data like block title, alignment etc
+- **fields** - all custom fields - also all the fields created in **ACF**
+
+finally you can create the template `template-parts/block/example-block.twig`:
+```twig
+{#
+/**
+ * Block Name: Example block
+ *
+ * This is the template that displays the example block.
+ */
+#}
+<div id="example-{{ block.id }}" class="wrapper">
+    <h1>{{ fields.title }}</h1>
+    <p>{{ fields.description }}</p>
+</div>
+<style type="text/css">
```

Comment:
I think inline styles are a good example. But I would also like to add a hint for how you can register your CSS files to work with Gutenberg so that CSS doesn’t have to necessarily be written into Twig files.

@pascalknecht has written a short tip for this: https://pascalknecht.ch/gutenberg-quick-tips-01-editor-styles/.